contributed_definitions/nyaml/NXphotoemission.yaml

```diff
@@ -0,0 +1,1641 @@
+        otherwise the local time zone is assumed per ISO8601.
+    end_time(NX_DATE_TIME):
+      exists: recommended
+      doc: |
+        Datetime of the end of the measurement.
+        Should be a ISO8601 date/time stamp. It is recommended to add an explicit time zone,
+        otherwise the local time zone is assumed per ISO8601.
+    method:
+      exists: recommended
+      doc: |
+        Name of the experimental method.
+        
+        If applicable, this name should match the terms given by `Clause 11`_ of
+        the `ISO 18115-1:2023`_ specification.
+        
+        Examples include:
+            * X-ray photoelectron spectroscopy (XPS)
+            * angle-resolved X-ray photoelectron spectroscopy (ARXPS)
+            * ultraviolet photoelectron spectroscopy (UPS)
+            * angle-resolved photoelectron spectroscopy (ARPES)
+            * hard X-ray photoemission spectroscopy (HAXPES)
+            * near ambient pressure X-ray photoelectron spectroscopy (NAPXPS)
+            * photoelectron emission microscopy (PEEM)
+            * electron spectroscopy for chemical analysis (ESCA)
+            * time-resolved angle-resolved X-ray photoelectron spectroscopy (trARPES)
+            * spin-resolved angle-resolved X-ray photoelectron spectroscopy (spin-ARPES)
+            * momentum microscopy
+        
+        .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
+        .. _Clause 11: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:sec:11
+    (NXuser):
+      exists: recommended
+      doc: |
+        Contact information of at least the user of the instrument or the investigator
+        who performed this experiment. Adding multiple users if relevant is recommended.
+      name:
+        doc: |
+          Name of the user.
+      affiliation:
+        doc: |
+          Name of the affiliation of the user at the time when the experiment was
+          performed.
+    (NXinstrument):
+      doc:
+      - |
+        Description of the MPES spectrometer and its individual parts.
+      - |
+        xref:
+          spec: ISO 18115-1:2023
+          term: 12.58
+          url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.58
+      energy_resolution(NXresolution):
+        exists: optional
+        doc:
+        - |
+          Overall energy resolution of the MPES instrument
+        - |
+          xref:
+            spec: ISO 18115-1:2023
+            term: 10.7 ff.
+            url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:10.7
+        - |
+          xref:
+            spec: ISO 18115-1:2023
+            term: 10.24
+            url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:10.24
+        physical_quantity:
+          enumeration: [energy]
+        type:
+          exists: recommended
+        resolution(NX_FLOAT):
+          unit: NX_ENERGY
```

Comment:
I think we should remove this here, or alternatively add all forseen resolution terms. It appears inconsistent